features/cookie-store.yml

```diff
@@ -1,35 +1,27 @@
-draft_date: 2024-09-09
-name: Cookie Store API
-description: TODO
+name: Cookie store
+description: To write...
```

Comment:
Do you want to take a crack at writing these descriptions? Or am I jumping the gun with a review and this was supposed to be a draft PR?

Reply:
I'm not good at writing descriptions, so I kind of left this as a request for assistance writing one!